tekton/cd/triggers/overlays/robocat/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+bases:
+- ../../base
+# patchesStrategicMerge:
+# -
```

Comment:
Well, originally the task did not support deploying a release for which there was no overlay defined. Now it does support that partially, so I could remove this.
Once I manager to work on having a central YAML file to track versions and overlays it will be a bit cleaner.